code/src/components/MazePage.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import labyrinth, { GameNextFetch } from 'reducers/labyrinth';
+import styled from 'styled-components/macro'
+import { ChildContainer, Button } from '../styles/GlobalStyles'
+
+const MazePage = () => {
+  const dispatch = useDispatch();
+  const description = useSelector((store) => store.labyrinth.status.description);
+  const actions = useSelector((store) => store.labyrinth.status.actions);
+  const onTypeButtonClick = (type, direction) => {
+    dispatch(GameNextFetch(type, direction));
+  }
+  const goToPreviousStatus = () => {
+    dispatch(labyrinth.actions.setPreviousStatus())
+  }
+
+  return (
+    <ChildContainer>
+      <div>
+        <GoBackButton type="button" onClick={() => goToPreviousStatus()}>Go back</GoBackButton>
+      </div>
+      <GameAlternatives>{description}</GameAlternatives>
+      {actions.length === 0 && (
+        <>
+          <lottie-player
+            src="https://assets4.lottiefiles.com/packages/lf20_zdouw3jj.json"
+            background="transparent"
+            speed="1"
+            style={{
+              width: '150px',
+              height: '150px'
+            }}
+            loop
+            autoplay />
+          <Button
+            type="button"
+            onClick={() => dispatch(labyrinth.actions.resetGame())}>
+            RESTART GAME
+          </Button>
+        </>)}
```

Comment:
This was nice! We had a component as last page. Also cool that you used the src for the lottie.